Remotion/Mixins/Core/Definitions/DependencyDefinitionBase.cs

```diff
@@ -62,12 +62,12 @@ public RequirementDefinitionBase RequiredType
       get { return _requirement; }
     }
 
-    public DependencyDefinitionBase Aggregator
+    public DependencyDefinitionBase? Aggregator
     {
       get { return _aggregator; }
     }
 
-    public string FullName
+    public string? FullName
```

Comment:
I think we should add a todo for FullName being null (here and in the other places where it's in the public API) Eventually, we will probably do an extension method and provide a sensible fallback for this case because it's just not helpful to have the metadata types return null for FullName.